cmd/launcher/launcher.go

```diff
@@ -610,6 +608,48 @@ func writePidFile(path string) error {
 	return nil
 }
 
+func initLauncherHistory(k types.Knapsack) {
+	// start counting uptime (want to reset on every run)
+	utcTimeNow := time.Now().UTC()
+	if err := k.LauncherHistoryStore().Set([]byte("process_start_time"), []byte(utcTimeNow.Format(time.RFC3339))); err != nil {
+		k.Slogger().Log(context.Background(), slog.LevelError,
+			"error setting process start time",
+			"err", err,
+		)
+	}
+
+	// set first recorded version (do not want to reset on every run)
+	installVersion, err := k.LauncherHistoryStore().Get([]byte("first_recorded_version"))
+	if err != nil {
```

Comment:
Worth logging what this error is? Same q for line 639/640 below

Reply:
yes, now logging